pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -377,9 +383,7 @@ type KubernetesSDConfig struct {
 	// Optional OAuth 2.0 configuration.
 	// Cannot be set at the same time as `authorization`, or `basicAuth`.
 	// +optional
-	OAuth2 *v1.OAuth2 `json:"oauth2,omitempty"`
-	// ProxyConfig allows customizing the proxy behaviour for this scrape config.
```

Comment:
Ah! In a previous PR it was suggested by @simonpasquier to remove the comments on in-line fields. I am not sure why though.

Reply:
got it, For inline fields it will show each field description  in API docs not the one added here https://github.com/prometheus-operator/prometheus-operator/blob/cbf0f66c687656b6a96e2acb8b618ef0aaaf138c/Documentation/api.md#kubernetessdconfig